sapphire/transformations/celestial.py

```diff
@@ -168,18 +184,22 @@ def equatorial_to_horizontal(latitude, longitude, timestamp, right_ascension,
 
     zenith, azimuth = horizontal_to_zenithazimuth(altitude, alt_azimuth)
 
-    return zenith, azimuth
+    return zenith, azimuth # NB: return is in zenithazimuth format not horizontal
```

Comment:
I'd say keep it but make it work as expected from the function name.
Someone can always manually use the `horizontal_to_zenithazimuth` to do the final conversion.
And this function works when not using astropy.